src/licensedcode/models.py

```diff
@@ -343,6 +343,10 @@ def validate(licenses, verbose=False, no_dupe_urls=False):
                 )
             if not lic.owner:
                 error('No owner')
+                
+            if lic.is_unknown:
+                if not "unknown" in lic.name:
```

Comment:
You should check in lic.key and not in lic.name, this is failing tests [here](https://dev.azure.com/nexB/scancode-toolkit/_build/results?buildId=4015&view=logs&j=a4427093-5c58-53ff-5e9f-a15ae68585b0&t=5e3c828d-e114-5dc5-9182-4c462c05ab86&l=1173)